extensions/blocks/membership-button/edit.jsx

```diff
@@ -0,0 +1,337 @@
+/**
+ * External dependencies
+ */
+
+import classnames from 'classnames';
+import SubmitButton from '../../utils/submit-button';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '../../utils/i18n';
+import { trimEnd } from 'lodash';
+import { getCurrencyDefaults } from '@automattic/format-currency';
+
+import {
+	Button,
+	PanelBody,
+	Placeholder,
+	Spinner,
+	TextControl,
+	withNotices,
+	SelectControl,
+} from '@wordpress/components';
+import { InspectorControls } from '@wordpress/editor';
+import { Fragment, Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { icon, SUPPORTED_CURRENCY_LIST } from '.';
+import { formatPrice } from '../simple-payments/utils';
```

Comment:
For this re-use to be more obvious and cleaner, let's move the `formatPrice` to [shared utils folder](https://github.com/Automattic/jetpack/tree/858b174e510ebcb59904cbeb0a89a072870d33e1/extensions/utils) instead of re-using it from another block's sources.

Reply:
Done!